contact.html

```diff
@@ -4,15 +4,47 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="stylesheet.css">
-    <title>Document</title>
+    <script src="https://kit.fontawesome.com/8d3908a90c.js" crossorigin="anonymous"></script>
+    <title>Final Website</title>
 </head>
 <body>
-    <nav>
-        <ul>
-          <li><a href="index.html">Home</a></li>
-          <li><a href="about.html">About Us</a></li>
-          <li><a href="contact.html">Contact Us</a></li>
-        </ul>
-      </nav>
+  <nav>
+    <div class="navbar">
+      <a href="https://www.seasmoss.com/"><img class="logo" src="images.img/Original Logo.png" alt="Logo"></a>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About Us</a></li>
+        <li><a href="contact.html">Contact Us</a></li>
+      </ul>
+    </div>
+    </nav>
+      <h1>Contact Us</h1>
+      <div class=" contactcontainer">
+        <form action="action_page.php">
+      
+          <label for="fname">First Name</label>
+          <input type="text" id="fname" name="firstname" placeholder="Your name..">
+      
+          <label for="lname">Last Name</label>
+          <input type="text" id="lname" name="lastname" placeholder="Your last name..">
+      
+          <label for="country">Country</label>
```

Comment:
Probably take out the Country or customize this a little more so that are understanding how forms and inputs work